File: Actividad 1/ejerciciorn5.py
import math #Importamos math para usar potencias

class ejerciciorn5: #Creamos la clase con el nombre "ejerciciorn5"

    #Inicializamos las variables
    suma = 0 
    x = 20
    y = 40

    #Realizamos los cálculos
    suma = suma + x
    x = x + math.pow(y, 2)
    suma = suma + (x/y)

    #Imprimimos por consola el resultado
    print("El valor de la suma es:", suma)

File: Actividad 1/ejerciciorn5.java
package main.java.ejercicioclase;

public class ejerciciorn5 {

    public static void main(String[] args) {

        double suma = 0;
        double x = 20;
        double y = 40;
        
        suma = suma + x;
        x = x + Math.pow(y, 2);
        suma = suma + (x / y);
        
        System.out.println("El valor de la suma es: " + suma);
        }
    }
